src/realm/sync/noinst/client_reset_operation.cpp

```diff
@@ -52,6 +52,17 @@ std::string ClientResetOperation::get_fresh_path_for(const std::string& path)
     return path + fresh_suffix;
 }
 
+bool ClientResetOperation::is_fresh_path(const std::string& path)
+{
+    const std::string fresh_suffix = ".fresh";
```

Comment:
can we add this as a const somewhere so we don't risk it going out of sync with `get_fresh_path_for` ?